User.py

```diff
@@ -48,14 +48,16 @@ def storeSession(self, run_name):
 
     def fetchNames(self):
         params = {"token": USER_TOKEN_STRING, "user": self.id}
-        response = requests.get("https://slack.com/api/users.info",
-                params=params)
-        user_obj = json.loads(response.text, encoding='utf-8')["user"]
+        response = requests.get("https://slack.com/api/users.info", params=params)
 
-        username = user_obj["name"]
-        real_name = user_obj["profile"]["real_name"]
+        parsed_message, isMessageOkay = parseSlackJSON(response)
+        if isMessageOkay:
```

Comment:
I've added a print to the parse method to print out the response status if it's not okay.,